pymc/tests/test_distributions.py

```diff
@@ -2279,27 +2289,25 @@ def test_dirichlet_multinomial_vectorized(self, n, a, extra_size):
         )
 
     @pytest.mark.parametrize(
-        "value,alpha,K,logp",
+        "alpha,K",
         [
-            (np.array([5, 4, 3, 2, 1]) / 15, 0.5, 4, 1.5126301307277439),
-            (np.tile(1, 13) / 13, 2, 12, 13.980045245672827),
-            (np.array([0.001] * 10 + [0.99]), 0.1, 10, -22.971662448814723),
-            (np.append(0.5 ** np.arange(1, 20), 0.5**20), 5, 19, 94.20462772778092),
-            (
-                (np.array([[7, 5, 3, 2], [19, 17, 13, 11]]) / np.array([[17], [60]])),
-                2.5,
-                3,
-                np.array([1.29317672, 1.50126157]),
-            ),
+            (0.5, 4),
+            (2, 12),
+            (np.array([0.5, 1.0, 2.0]), 3),
+            (np.arange(1, 7, dtype="float64").reshape(2, 3), 5),
         ],
     )
-    def test_stickbreakingweights_logp(self, value, alpha, K, logp):
-        with Model() as model:
+    def test_stickbreakingweights_logp(self, alpha, K, _compile_stickbreakingweights_logpdf):
```

Comment:
Is it acceptable to combine `test_stickbreakingweights_logp` and `test_stickbreakingweights_vectorized` in a single test?

Reply:
If `test_stickbreakingweights_vectorized` fails, it would point to shapes not being handled properly to form batches. So, lets keep them separate to have better isolation of tests.

Reply:
Now I separate the test for batched alpha